commands/ssubscribe.md

```diff
@@ -0,0 +1,17 @@
+Subscribes the client to the specified shard channels.
+
+In a Redis cluster, shard channels are hashed to a slot and client can subscribe to a node covering a slot (primary/replica)  to recieve the messages published. 
+ 
+Once the client enters the subscribed state it is not supposed to issue any other commands, except for additional `SUBSCRIBE`, `SSUBSCRIBE`, `PSUBSCRIBE`, `UNSUBSCRIBE`, `PUNSUBSCRIBE`, `PING`, `RESET` and `QUIT` commands.
```

Comment:
I don't think this is true any more for RESP3, as messages are sent out of band.